jabgui/src/main/java/org/jabref/gui/consistency/ConsistencyCheckAction.java

```diff
@@ -55,10 +53,11 @@ public BibliographyConsistencyCheck.Result call() throws Exception {
                 if (databaseContext.isEmpty()) {
                     throw new IllegalStateException((Localization.lang("No library present")));
                 }
-                List<BibEntry> entries = databaseContext.get().getEntries();
+
+                BibDatabaseContext bibContext = databaseContext.orElseThrow(() -> new IllegalStateException("database not present"));
```

Comment:
Never throw `IllegalStateException`, because this tears down JabRef. Just use `.get()` here.